### DISCUSSION TITLE
Inter bold not working when printing

### DISCUSSION THREAD

**Discussion by henri-brasseur** [rsms/inter#751] [Q&A] [answered]
Hello,

We have a customer who is apparently using your font for their documents and it seems that, when it comes to printing the font, the bold effect is not taken into consideration.

I happened to discover this github repository and i see that this font is mainly for screens. I am no expert in fonts dev, so i was wondering if you could help me figure out why this problem exist and if this is a normal behaviour ?

thanks in advance !

**Comment by kenmcd** [comment]
Not likely an issue with the font.
Most likely user error.
Which may be possible to fix with actual specific information. 
Right now this is an "it don't work" post which is useless for troubleshooting.

**Reply by henri-brasseur** [reply]
oh, i'm sorry...
did a test with the font on Microsoft Word (o365)
Used Inter Variable Text, sample text on 4 different lines.
First line : Bold
Second line : normal
Third line : bold italic
4th line : italic

Printed out, and the lines presumably bold were not

second test, as i see that you have a different font to install to have a more bold font, i used the same text, with the following fonts :
First line : extra-bold
Second line : normal
Third line : extra-bold italic
4th line : italic

Printed out, same result, the text supposedly bold isn't bold on paper.

**Reply by henri-brasseur** [reply]
tested on the customer's computer, and customer's printer as well as my computer and printer in another location, same results

**Reply by kenmcd** [reply, marked as answer]
I no longer have access to O365, but I did test in Office 2024 
(which is basically the same now)

Word does not export variable fonts to PDF correctly, but printing should work.
This image below was printed from Word 2024 on Windows 11.
I used the variable fonts from this repo (not GF), and replicated your test above.

![Microsoft Word - Inter Text Variable Tests docx cropped](https://github.com/user-attachments/assets/cfc10bc6-0d7c-4c81-be47-d1a778bea402)

My guess is you and your client have some name conflicts from having different versions of the fonts installed - old and new, VFs and statics, GF vs. repo.
And this can cause odd behavior, like what you are describing.
The GF versions, and the versions from this repo have different names.
And some of those names are duplicates and will cause conflicts.

My suggestion is to use the only the static fonts and get that working.
Un-install all old Inter fonts.
O365 did have Inter in the Cloud Fonts so that could also cause a conflict.
Check the Windows CloudFont folder and delete the `Inter` folder if there is one.

GF static fonts include three optical sizes, and they are different family names. 
`Inter_18pt`, `Inter_24pt`, and `Inter_28pt` - 18 styles in each.

The static fonts from this repo have two optical sizes.
`Inter` is opsz 14pt, and `Inter Display` is opsz 32pt (I think).

Because they do have different family names you could have them all installed without conflicts.

So un-install all Inter fonts.
Then install only the statics.
And see how that works.

**Reply by henri-brasseur** [reply]
Hello, indeed, only using the static solved the issue ! Thanks :)

**Reply by kenmcd** [reply]
> Hello, indeed, only using the static solved the issue ! Thanks :)

Great.
You could also install the variable fonts from here in the repo because it has a different family name - Inter Variable Text.

The GF variable family name, Inter, will conflict with the repo statics family name, which is also just Inter.
So you cannot not use the GF variable with the repo statics, or any of the older version statics.
Which is probably what happened - name conflicts.
